src/components/Heading/components/Identity/Identity.tsx

```diff
@@ -182,18 +171,23 @@ export const Identity = () => {
                 <input
                   type='text'
                   name={field.name}
-                  value={values[field.name]}
+                  value={(values as any)[field.name]}
```

Comment:
Why do you need to cast to any? I think the values should have a strict type

Reply:
Updated. 👍